getting-started/customize-dev-portal-example.md

```diff
@@ -0,0 +1,230 @@
+# Integrate a plugin into your Tanzu Developer Portal
+
+This topic guides you through integrating the hello-world plugin into your Tanzu Developer Portal by using the configurator tool.
+
+The hello-world plugin has been created a demonstration of the configurator tool's capabilities and does not carry a standalone valuable function.
+
+For general information on the Tanzu Developer Portal configurator and its basic concepts, please refer to [Tanzu Developer Portal Configurator](../tap-gui/configurator/about.hbs.md)
+
+## <a id="prereqs"></a>Prerequisites
+
+Before you start, you must have:
+
+- Tanzu Application Platform cluster with the full profile
+- Tanzu Developer Portal Configurator bundle available (`tpb.tanzu.vmware.com`). 
+
+To verify it is present, run the following command: 
+```console
+tanzu package available list --namespace tap-install | grep tpb.tanzu.vmware.com
+```
+- Instance of the canonical Tanzu Developer Portal.
+
+To get the fully-qualified domain name for the portal, run the following command:
+```console
+kubectl get httpproxy tap-gui -n tap-gui
+```
+> **Note:** by default, Tanzu Developer Portal uses a self-signed certificate and may result in a security error in the browser. To eliminate that error message, the Platform Engineer shall need to configure SSL. For more information, please refer to [Configure a TLS certificate by using an existing certificate](../tap-gui/tls/enable-tls-existing-cert.hbs.md).
+
+- At least one configured developer namespace
+- At least one operational Supply Chain
+
+To confirm the last two prerequisites, it is recommended to deploy a demo workload as described in [Deploy an app on Tanzu Application Platform](./deploy-first-app.hbs.md). 
+
+
+## <a id="you-will"></a>What you will do
+
+- Create a customized Tanzu Developer Portal containing the hello-world plugin
+- Observe the customized Tanzu Developer Portal instance
+
+## <a id="customize-dev-portal"></a>Customize your Tanzu Developer Portal by adding the hello-world plugin
+
+Complete the following steps to integrate the hello-world plugin into your Tanzu Developer Portal.
+
+### Procedure
+
+1. Create the **tpb-config.yaml** file specifying the list of additional plugins that you want to integrate. In this example, we only need to specify the hello-world plugin that has both frontend and backend parts:
+
+    ```yaml
+    app:
+      plugins:
+        - name: '@tpb/plugin-hello-world'
+          version: '^1.6.0-release-1.6.x.1'
+    backend:
+      plugins:
+        - name: '@tpb/plugin-hello-world-backend'
+          version: '^1.6.0-release-1.6.x.1'
+    ```
+> **Note:** If the plugin's version is not specified, it is likely that the workload that uses the config file as input will fail.
```

Comment:
I tried it for TAP 1.6.1 (without any rc) - it did work with the version, but didn't try it without the version... If no longer needed, that is great.

Reply:
@chinigo i didn't try it with verdaccio but when we omit the version when using our artifactory, we have some issues indeed because some plugins seem to have version numbers that are out of range and that appear as most recent when running `yarn info` (i remember having seen some plugin with a "0.2.4" version)

Reply:
Ok, thanks Fouad. @andreizimin I'd leave this exactly as you have it, then. Let's be conservative and keep the version numbers.